.yamato/all-universal_split.yml

```diff
@@ -0,0 +1,111 @@
+ 
+# This file is generated by .yamato/ruamel/build.py. Don't edit this file manually. 
+# Introduce any changes under .yamato/config/*.metafile files (for most cases) or under .yamato/ruamel/* within Python (more elaborate cases), and rerun build.py to regenerate all .yml files. 
+# Read more under .yamato/docs/readme.md 
+
+{% metadata_file .yamato/_latest_editor_versions_trunk.metafile -%}
+
+---
+
+PR_Universal_Split_trunk:
+    name: Universal_Split PR Job - trunk
+    dependencies:
+      -  path: .yamato/all-urp_2d.yml#PR_URP_2D_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-urp_foundation.yml#PR_URP_Foundation_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-urp_lighting.yml#PR_URP_Lighting_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-urp_postpro.yml#PR_URP_PostPro_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-urp_terrain.yml#PR_URP_Terrain_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-universal_stereo.yml#PR_Universal_Stereo_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-universal_hybrid.yml#PR_Universal_Hybrid_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-vfx_urp.yml#PR_VFX_URP_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-shadergraph.yml#PR_ShaderGraph_trunk
+         rerun: on-new-revision
+    variables:
+        UTR_VERSION: "current"
+        TEST_FILTER: .*
+Nightly_Universal_Split_trunk:
```

Comment:
Should be called by https://github.com/Unity-Technologies/Graphics/blob/universal/split-project-20.2-autosplit/.yamato/_abv.yml#L27 (currently, I don't see any trigger for this nightly pipeline)